sample-tests-and-e2e-tests/src/use-effect/use-effect-component-with-props.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useState } from 'react'
+import { expensiveOperation1, expensiveOperation2, expensiveOperation3, expensiveOperation4 } from './expensiveOperations';
+
+export default function ComponentWithProps({ strProp, arrProp }) {
```

Comment:
Is it possible to make it consistent with another test component and use function declaration as opposed to a function statement?

Reply:
Sure